ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1479,36 +1479,11 @@ void Plane::mavlink_delay_cb()
     DataFlash.EnableWrites(true);
 }
 
-/*
- *  send data streams in the given rate range on both links
- */
-void Plane::gcs_data_stream_send(void)
-{
-    gcs().data_stream_send();
-}
-
-/*
- *  look for incoming commands on the GCS links
- */
-void Plane::gcs_update(void)
-{
-    gcs().update();
-}
-
 /*
   send airspeed calibration data
  */
 void Plane::gcs_send_airspeed_calibration(const Vector3f &vg)
 {
-    gcs().send_airspeed_calibration(vg);
```

Comment:
Pretty sure this was slightly to aggressive a trim?

Reply:
Gah.

I was going to move that call into the library, decided against it... and missed...